### PR TITLE
30 Make geotiffs right side up

### LINKED ISSUE #30: BUG: Make geotiffs right side up
# Requested Update

### Description
*  Currently geotiffs are upside down. Flip them over.​

### Background and Motivation
*  We do not want upside down GeoTIFFs. It will confuse people.
* imagery_annotated opposite geotiff output viewed in Google Earth​

### Alternative Solutions
*  Put a warning that the user must view geotiffs upside down...​
* Rotate monitors
  
### Environment
* geoips repo, v1.4.8+​

### Code to demonstrate issue
image

*  $GEOIPS/tests/scripts/ahi.WV.geotiff.sh will produce above upside down geotiff.​

 @mindyls ADD PICTURE!

### Checklist for Completion
* [  ] GeoTIFFS are right side up.

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist

* [x] Required ***existing tests*** pass (ahi geotiff test)
* [x] NO required ***unit tests*** (unit tests coming soon!)
* [x] NO Required ***integration tests*** (tested with existing tests)
* [x] Required ***documentation*** added for new/modified functionality
* [x] Required ***release notes*** added for new/modified functionality

# Related Issues
fixes #30 

# Testing Instructions
To test this branch you can run ./tests/scripts/ahi.tc.WV.geotiff.sh, which will output 0. Also, the way I ensured the Geotiffs were now correct, was by running `conda install -c condo-forge gdal`, and once that was installed, you can run the command `gdalinfo <path_to_geotiff>` which will produce the following output.

Screenshot 2023-10-05 at 10 08 46 AM

As you can see above, the corner coordinates are correct for each corner, whereas in the previous version Upper Left was actually below Lower Left, causing the flip.

# Summary
This branch fixes the upside down geotiffs bug that was occurring when loading the .tif into other softwares, such as Google Earth Pro. To do this, the following files were modified.

  - geoips/geoips/plugins/modules/output_formatters/geotiff_standard.py
  - geoips/tests/outputs/ahi.tc.WV.geotiff/20200405_000000_SH252020_ahi_himawari-8_WV_100kts_100p00_1p0.tif

# Output
New output in Google Earth Pro

Screenshot 2023-10-05 at 10 11 58 AM

### CODE REVIEW

**Review by mindyls — DISMISSED** [review]
Yay!

05_094820 single_source.py:1874 INTERACTIVE:     SINGLESOURCESUCCESS ${GEOIPS_OUTDIRS}/preprocessed/tcwww/tc2020/SH/SH252020/tif/WV/himawari-8/20200405_000000_SH252020_ahi_himawari-8_WV_100kts_100p00_1p0.tif
05_094820 single_source.py:1884 INTERACTIVE: READER_NAME: ahi_hsd
05_094820 single_source.py:1885 INTERACTIVE: PRODUCT_NAME: WV
05_094820 single_source.py:1886 INTERACTIVE: NUM_PRODUCTS: 1
05_094820 single_source.py:1887 INTERACTIVE: NUM_DELETED_PRODUCTS: 0
05_094820 run_procflow.py:70   INTERACTIVE: Completed geoips PROCFLOW single_source processing, done!
05_094820 run_procflow.py:76   INTERACTIVE: Total time: 0:00:13.646767
05_094820 run_procflow.py:96   INTERACTIVE: Return value: 0

File: geoips/plugins/modules/output_formatters/geotiff_standard.py [hunk 1]
```diff
@@ -90,15 +90,15 @@ def call(
...
             res_deg_y = (lats[0][0] - lats[-1][-1]) / height
 
-            minlat = area_def.area_extent_ll[1]
+            maxlat = area_def.area_extent_ll[-1]
             minlon = area_def.area_extent_ll[0]
 
             transform = Affine.translation(
-                minlon - res_deg_y / 2, minlat - res_deg_x / 2
-            ) * Affine.scale(res_deg_y, res_deg_x)
+                minlon - res_deg_x / 2, maxlat - res_deg_y / 2
```
Comment (mindyls) [inline geoips/plugins/modules/output_formatters/geotiff_standard.py:97]: Maybe add a quick note as to what this is doing? (documentation!)

**Review by mindyls — DISMISSED** [review]
Added one note.  I might be fine leaving it though to avoid re-approving and rerunning tests  :)

File: geoips/plugins/modules/output_formatters/geotiff_standard.py (hunk 1 above)
Comment (evrose54) [reply geoips/plugins/modules/output_formatters/geotiff_standard.py:97]: I saw your comment below and will leave it as is for the time being.

**Review by cpcamacho — APPROVED** [review]

File: geoips/plugins/modules/output_formatters/geotiff_standard.py (hunk 1 above)
Comment (mindyls) [reply geoips/plugins/modules/output_formatters/geotiff_standard.py:101]: Well, I decided to add a note, for future reference and sanity.. Is that correct, @evrose54 ?

**Review by cpcamacho — APPROVED** [review]

**Review by mindyls — APPROVED** [review]
@evrose54 assuming my comment in the code is correct, this is good to go.  Resolve if you're ok with it

File: geoips/plugins/modules/output_formatters/geotiff_standard.py (hunk 1 above)
Comment (evrose54) [reply geoips/plugins/modules/output_formatters/geotiff_standard.py:101]: Yup, your comment is correct @mindyls . I found another conversation on GitHub which pointed me in the right direction to fix this issue. https://github.com/rasterio/rasterio/issues/2565

**Review by jsolbrig — DISMISSED** [review]

**Review by mindyls — DISMISSED** [review]
Good to go, thanks for resolving conversation Evan!

**Comment by evrose54** [comment]
Of course Mindy!

**Comment by AJThorpe** [comment]
- [x] AHI TC test: produces proper output & orientation 

Works for me! Approved review.

**Review by AJThorpe — DISMISSED** [review]

**Review by mindyls — APPROVED** [review]
Resolving release note conflict cleared the reviews.  I am going to merge with just my final approval.

**Review by cpcamacho — APPROVED** [review]

**Pull request merged**